src/lib.rs

```diff
@@ -2692,6 +2692,79 @@ pub unsafe trait FromBytes: FromZeros {
         mut_from_prefix_suffix(bytes, None, CastType::Suffix).map(swap)
     }
 
+    /// Interprets the given `bytes` as a `&Self` with a DST length equal to
+    /// `count`.
+    ///
+    /// This method verifies that `bytes.len() == size_of::<T>() * count` and
+    /// that `bytes` is aligned to `align_of::<T>()`. It also ensures that
+    /// `sizeof::<T>() * count` does not overflow a `usize`. If any of the
+    /// length, alignment, or overflow checks fail, it returns `Err`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use zerocopy::FromBytes;
+    /// # use zerocopy_derive::*;
+    ///
+    /// # #[derive(Debug, PartialEq, Eq)]
+    /// #[derive(FromBytes, Immutable)]
+    /// #[repr(C)]
+    /// struct Pixel {
+    ///     r: u8,
+    ///     g: u8,
+    ///     b: u8,
+    ///     a: u8,
+    /// }
+    ///
+    /// let bytes = &[0, 1, 2, 3, 4, 5, 6, 7][..];
+    ///
+    /// let pixels = <[Pixel]>::ref_from_with_trailing_elements(bytes, 2).unwrap();
+    ///
+    /// assert_eq!(pixels, &[
+    ///     Pixel { r: 0, g: 1, b: 2, a: 3 },
+    ///     Pixel { r: 4, g: 5, b: 6, a: 7 },
+    /// ]);
+    ///
+    /// ```
+    ///
+    /// Since an explicit `count` is provided, this method supports types with
+    /// zero-sided trailing slice elements. Methods such as [`ref_from`]
```

Comment:
```suggestion
    /// zero-sized trailing slice elements. Methods such as [`ref_from`]
```

(And elsewhere while you're at it - looks like a typo snuck in to `main`)

Reply:
Done